### PR TITLE
Lower LightGBM test AUC base value

### PR DESCRIPTION
**Pull request by simonzhaoms**
### Description

The new LightGBM from SynapseML gives lower AUC value which makes smoke tests failed.
This PR lowers the test base value.

### Related Issues

See https://github.com/microsoft/recommenders/issues/1615

### Checklist:

- [X] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [X] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [X] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: tests/smoke/examples/test_notebooks_pyspark.py [hunk 1]
```diff
@@ -59,4 +59,4 @@ def test_mmlspark_lightgbm_criteo_smoke(notebooks, output_notebook, kernel_name)
     results = sb.read_notebook(output_notebook).scraps.dataframe.set_index("name")[
         "data"
     ]
-    assert results["auc"] == pytest.approx(0.68895, rel=TOL, abs=ABS_TOL)
+    assert results["auc"] == pytest.approx(0.65, rel=TOL, abs=ABS_TOL)
```
Comment (miguelgfierro) [inline tests/smoke/examples/test_notebooks_pyspark.py:62]: this error is weird, we have been running this notebook for a couple of years, maybe there are some default changes in the new version of LightGBM? I ping Mark and Ilia in the issue, let's see if they reply before making the change

**Review by miguelgfierro — CHANGES_REQUESTED** [review]

**Review by anargyri — APPROVED** [review]

**Comment by miguelgfierro** [comment]
@anargyri the change made than Ilia requires to also change the result of AUC, meaning, do we still need to merge this PR?

**Review by miguelgfierro — APPROVED** [review]
Approved: waiting for the decision of Ilia and Andreas

**Review by imatiach-msft — APPROVED** [review]

**Comment by anargyri** [comment]
> @anargyri the change made than Ilia requires to also change the result of AUC, meaning, do we still need to merge this PR?

Yes, I think we need to merge this PR. The other PR (early stopping parameter) does not affect AUC at all.

**Pull request merged**